modules/client/rendering/build.gradle

```diff
@@ -0,0 +1,5 @@
+version = getSubprojectVersion(project, "1.0.0")
+
+dependencies {
+	implementation project(path: ":legacy-fabric-api-base", configuration: "dev")
```

Comment:
This should use `moduleDependencies`
Ditto for all other subprojects

Reply:
next pr